javascript-loops-practice/filter-out-strings.js

```diff
@@ -2,9 +2,19 @@
 function filterOutStrings(values) {
   var newArr = [];
   for (let i = 0; i < values.length; i++) {
-    if (typeof values[i] === 'number') {
+    if (typeof values[i] === 'string') {
+      continue;
+    } else {
```

Comment:
Nice!! The pseudocode looks great and matches your function perfectly 🤓 👌 

However, the `continue` keyword is often an indicator that you have your logic backwards. If your work for a condition is to "move on", odds are you should really be looking for the opposite. This function will be more straightforward If your condition was `typeof values[i] !== 'string'`

Reply:
Ohhh that makes perfect sense. Got it!!

Reply:
Is it okay to leave a code block completely empty?